crates/router/src/connector/mollie/transformers.rs

```diff
@@ -298,21 +364,48 @@ pub struct BankDetails {
 
 pub struct MollieAuthType {
     pub(super) api_key: String,
+    pub(super) key1: String,
 }
 
 impl TryFrom<&types::ConnectorAuthType> for MollieAuthType {
     type Error = Error;
     fn try_from(auth_type: &types::ConnectorAuthType) -> Result<Self, Self::Error> {
-        if let types::ConnectorAuthType::HeaderKey { api_key } = auth_type {
+        if let types::ConnectorAuthType::BodyKey { api_key, key1 } = auth_type {
```

Comment:
this has to be notified to dashboard team. @srujanchikke  please take care of this